test/create-open.test.js

```diff
@@ -340,5 +337,49 @@ Object.keys(testAPIs).forEach(API => {
         assert.equal(res[1].payload.value, 'hello2')
       })
     })
+
+    describe("Close", function() {
+      beforeEach(async () => {
+        Object.values(orbitdb.caches).forEach(cache => {
+          cache.handlers = new Set()
+        })
+      })
+
+      it('closes a custom store', async () => {
+        const directory = path.join(dbPath, "custom-store")
+        db = await orbitdb.open('xyz', { create: true, type: 'feed', directory })
+        await db.close()
+        assert.strictEqual(db._cache._store._db.status, 'closed')
+      })
+
+      it('successfully manages multiple caches', async() => {
+        // Cleaning up cruft from other tests
+        const directory = path.join(dbPath, "custom-store")
+        const directory2 = path.join(dbPath, "custom-store2")
+
+        const db1 = await orbitdb.open('xyz1', { create: true, type: 'feed', })
+        const db2 = await orbitdb.open('xyz2', { create: true, type: 'feed', directory })
+        const db3 = await orbitdb.open('xyz3', { create: true, type: 'feed', directory  })
+        const db4 = await orbitdb.open('xyz4', { create: true, type: 'feed', directory: directory2  })
+        const db5 = await orbitdb.open('xyz5', { create: true, type: 'feed', })
+
+        await db1.close()
+        await db2.close()
+        await db4.close()
+
+        assert.strictEqual(orbitdb.cache._store._db.status, 'open')
+        assert.strictEqual(db2._cache._store._db.status, 'open')
```

Comment:
Shouldn't this be 'closed'?

Reply:
I believe this is correct as it is. `db2` and `db3` use the same cache (`directory`), so if `db3` is not closed yet, closing `db2` should keep the cache open for `db3` to keep using it. Makes sense?